Dockerfile-client

```diff
@@ -17,5 +18,10 @@ RUN yarn run build
 RUN echo '*/1 * * * * cd /airnode && yarn run dev:invoke' > local-cron
 RUN /usr/bin/crontab /airnode/local-cron 
 
+HEALTHCHECK --interval=5m --timeout=2m \
+    CMD (strace -p $(pgrep crond) -s9999 -o cronLog.txt) & sleep 70 ; kill $! \
+        && cat cronLog.txt | grep crond > /dev/null; if [ 0 != $? ]; then kill 1; fi && rm cronLog.txt
+
 CMD cp out/config.json /airnode/packages/node/__dev__/config.json | true \
+    && cp out/.env /airnode/packages/node/__dev__/.env | true \
```

Comment:
Storing the config.json and .env into /airnode/out creates this error:
```
cp: can't stat 'out/config.json': No such file or directory
cp: can't stat 'out/.env': No such file or directory
```
Finally to make it working, I stored the .env and the config.json in the root directory of airnode, so I think this command need to be changed if we want to use a different directory than the root

Reply:
the original [dockerfile ](https://github.com/api3dao/airnode/blob/master/Dockerfile)uses the config.json in the root so I followed the same design choice.

Reply:
My question is why the "out" folder when the docker file read then from the root.
I would remove from both if the docker finally expect them in the root folder.

Reply:
when you run the
```
docker run -it --init --rm \
    -v $(pwd):/airnode/out \
    api3/airnode-client:pre-alpha
```

it mounts the current working directory to airnode/out . So essentially when you run it from the root, it mounts the root folder to airnode/out in the docker container. So now config.json is in the airnode/out folder in the docker container. 

> Have you run it locally? Are you storing the files in the our folder or in the root?

 Yes I have runned it locally, I was storing the files in the root. If you wanted to save it in a out folder you could modify the run command like so 
 ```
 docker run -it --init --rm \
    -v $(pwd)/out:/airnode/out \
    api3/airnode-client:pre-alpha
 ```
 
and that should work, but I haven't tested it.

Reply:
I would change the docker run command. It will me more clear to the user that has to store the config.json and the .env file in the out folder, but if you don't think/want to, we can merge this PR as it is and create another issue to address the out directory thing